conf/messages.en

```diff
@@ -317,6 +317,7 @@ supplementary.consignor.eori = Exporter’s EORI number
 supplementary.consignor.address.info = Exporter’s name and business address
 supplementary.exporter-details.help-item1 = Exporter’s EORI number, go to data element 3/2 Exporter Identification Number
 supplementary.exporter-details.help-item2 = Exporter’s name and business address, go to data element 3/1 Exporter
+supplementary.exporter-details-name-address = Exporter’s name and address
```

Comment:
For all new messages it's better to use `declaration.exporter-details` etc. Supplementary is old thing when we had just one journey. During Changing the page we're trying to get rid of `supplementary` and replace it with `declaration`.